components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -183,6 +183,7 @@ interface PageHandler {
   ClearConversationHistory();
   RetryAPIRequest();
   GetAPIResponseError() => (APIError error);
+  ClearErrorAndGetFailedMessage() => (ConversationTurn turn);
```

Comment:
need comment

Reply:
added comment

Reply:
...and removes it from the history... (not clear from the function name)